seed/utils/merge.py

```diff
@@ -57,6 +57,8 @@ def merge_properties(state_ids, org_id, log_name, ignore_merge_protection=False)
         merged_state = merge_ubid_models([state_1.id, state_2.id], merged_state.id, PropertyState)
 
         views = PropertyView.objects.filter(state_id__in=[state_1.id, state_2.id])
+        if not views:
+            return
```

Comment:
You can get ther via the "Trigger Matching and Merging" button if 
- the ubid threshold has been loosened from something like 1.0 to 0.1
- there are existing properties that now meet the matching criteria

Here's my hunch. As we iterate over unmatched_states in the merge_unmatched_states function, we start with `state_1`. If `state_1` matches with `state_2`, they are combined into a new state. This new state gets associated with the relevant view, while `state_1` and `state_2` are disassociated from their views. When we later iterate over `state_2`, it will match with `state_1` again. However, since `state_1` and `state_2` have already been merged and disassociated from their views, this triggers the if statement.